apps/meteor/app/api/server/v1/chat.ts

```diff
@@ -822,3 +823,30 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'chat.getURLPreview',
+	{ authRequired: true },
+	{
+		async get() {
+			const { roomId, url } = this.queryParams;
+
+			if (!roomId) {
```

Comment:
I saw you created the `ajv` schema for this endpoint but is not using it and you're checking the params by hand like this one.

I'd say to use the `ajv` function `isChatGetURLPreviewProps` to validate the endpoints params instead of checking them manually 😉

Reply:
Oopsss :grimacing: thanks for noticing that. Should be fixed now